src/connectors/index.tsx

```diff
@@ -23,6 +29,26 @@ export const injected = new InjectedConnector({
   supportedChainIds: [1], // we only support mainnet
 });
 
+// export const bitski = new BitskiConnector({
+//   clientId: BITSKI_KEY,
+//   redirectUri: 'https://localhost:3000/bitski-callback.html',
+// })
```

Comment:
what's the plan with bitski? if you don't plan to use the commented code can you remove it?